crates/graphql/src/schema.rs

```diff
@@ -27,18 +27,18 @@ struct Creator {
 }
 
 #[derive(Debug, Clone, Copy)]
-pub struct Lamports(u64);
+pub struct UnsignedInt8(u64);
 
-#[juniper::graphql_scalar(description = "Lamports")]
-impl<S> GraphQLScalar for Lamports
+#[juniper::graphql_scalar(description = "UnsignedInt8")]
+impl<S> GraphQLScalar for UnsignedInt8
```

Comment:
If we switch activated_at to date time scaler can this stay as a lamports scalar.

Reply:
yes